app/views/admin/orders/_order.html.erb

```diff
@@ -4,14 +4,45 @@
   <td><%= order.phone_num %></td>
   <td><%= order.address %></td>
   <td><%= order.note %></td>
-  <td><%= order.status %></td>
+  <td>
+    <%= form_for([:admin,order], html: {:onsubmit => "return confirm('Are you sure?');" }) do |f| %>
```

Comment:
`:onsubmit => "return confirm('Are you sure?');"`
-> xử lý sự kiện trong file js nhé